### ISSUE TITLE
tx-sign panic

### ISSUE THREAD

**Issue by blocklee** [Qitmeer/qng#263]
TnEvLExwzew6LPL13yXmWKnnZxD1c5Lr8Tw has 76.0073 Meer  which are composed of two utxos: 
- 8a29926f23de9037aa267207dc439749129591993c74cde45504a9b816b6a4c5:1
```bash
{
  "coin": "MEER Asset",
  "amount": 100730000,
  "scriptPubKey": {
    "asm": "OP_DUP OP_HASH160 2421972d46cedd2283d244665a05bde2aec3446f OP_EQUALVERIFY OP_CHECKSIG",
    "hex": "76a9142421972d46cedd2283d244665a05bde2aec3446f88ac",
    "reqSigs": 1,
    "type": "pubkeyhash",
    "addresses": [
      "TnEvLExwzew6LPL13yXmWKnnZxD1c5Lr8Tw"
    ]
  }
}
```
- 7c3e320bca2d9fb52963c317bbd029ae4a6dfcaf404f7ed24394ede008dd5055:3
```bash
{
  "coin": "MEER Asset",
  "amount": 7500000000,
  "scriptPubKey": {
    "asm": "6ef56063 OP_CHECKLOCKTIMEVERIFY OP_DROP OP_DUP OP_HASH160 2421972d46cedd2283d244665a05bde2aec3446f OP_EQUALVERIFY OP_CHECKSIG",
    "hex": "046ef56063b17576a9142421972d46cedd2283d244665a05bde2aec3446f88ac",
    "reqSigs": 1,
    "type": "cltvpubkeyhash",
    "addresses": [
      "TnEvLExwzew6LPL13yXmWKnnZxD1c5Lr8Tw"
    ]
  }
}
```

Now create a transaction to organize these utxo:

```bash
./qx tx-encode -i 8a29926f23de9037aa267207dc439749129591993c74cde45504a9b816b6a4c5:1:4294967295:TxTypeRegular -i 7c3e320bca2d9fb52963c317bbd029ae4a6dfcaf404f7ed24394ede008dd5055:3:4294967295:TxTypeRegular -l 0 -o TnEvLExwzew6LPL13yXmWKnnZxD1c5Lr8Tw:76.0072:0:TxTypeRegular
0100000002c5a4b616b8a90455e4cd743c99919512499743dc077226aa3790de236f92298a01000000ffffffff5550dd08e0ed9443d27e4f40affc6d4aae29d0bb17c36329b59f2dca0b323e7c03000000ffffffff01000080c809c5010000001976a9142421972d46cedd2283d244665a05bde2aec3446f88ac000000000000000093836263020000-7b22696e707574223a7b2230223a302c2231223a307d2c226f7574707574223a7b2230223a307d7d

{
  "txid": "53fe6c4e7ae7ab6d24caa1179ed8cb21599072f67f8ec204a550efa9c7746ffd",
  "txhash": "cfc73ce4d1ff71ad90ac748c303053384bc71c91c886bab303b1d005c754e041",
  "version": 1,
  "locktime": 0,
  "expire": 0,
  "vin": [
    {
      "type": "TxTypeRegular",
      "scriptSig": {
        "asm": "",
        "hex": ""
      }
    },
    {
      "type": "TxTypeRegular",
      "scriptSig": {
        "asm": "",
        "hex": ""
      }
    }
  ],
  "vout": [
    {
      "coin": "MEER Asset",
      "amount": 7600720000,
      "scriptPubKey": {
        "asm": "OP_DUP OP_HASH160 2421972d46cedd2283d244665a05bde2aec3446f OP_EQUALVERIFY OP_CHECKSIG",
        "hex": "76a9142421972d46cedd2283d244665a05bde2aec3446f88ac",
        "reqSigs": 1,
        "type": "pubkeyhash",
        "addresses": [
          "TnEvLExwzew6LPL13yXmWKnnZxD1c5Lr8Tw"
        ]
      }
    }
  ]
}

```

But, a panic occur when sign the tx:

```bash
./qx tx-sign -k e2ec07936723d6b8c054f1f6bfe2cf1c439733303e5a6f0062d54168d9265b14 -n testnet 0100000002c5a4b616b8a90455e4cd743c99919512499743dc077226aa3790de236f92298a01000000ffffffff5550dd08e0ed9443d27e4f40affc6d4aae29d0bb17c36329b59f2dca0b323e7c03000000ffffffff01000080c809c5010000001976a9142421972d46cedd2283d244665a05bde2aec3446f88ac000000000000000093836263020000-7b22696e707574223a7b2230223a302c2231223a307d2c226f7574707574223a7b2230223a307d7d
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/Qitmeer/qng/qx.TxSign({0xc0005eb470, 0x1, 0x40c3380?}, {0x7ff7bfeff364?, 0xc0005eb428?}, {0x7ff7bfeff35c, 0x7})
	/Users/blocklee/git/node/code/qng/qx/handle_tx.go:124 +0x4df
github.com/Qitmeer/qng/qx.TxSignSTDO({0x7ff7bfeff318?, 0xc000012090?}, {0x7ff7bfeff364?, 0x4623fa8?}, {0x7ff7bfeff35c?, 0x462fb3c?})
	/Users/blocklee/git/node/code/qng/qx/handle_tx.go:225 +0x5b
main.main()
	/Users/blocklee/git/node/code/qng/cmd/qx/qx.go:1419 +0x72a5
```

**Comment by jamesvan2019** [comment]
Fixed By#265

**Comment by dindinw** [comment]
fixed by #264 #265 #266
